package.json

```diff
@@ -2,6 +2,11 @@
   "name": "@openedx/brand-openedx",
   "version": "1.0.0-semantically-released",
   "description": "The default branding and SASS theme package containing for Open edX applications. This package is designed to be copied and customized.",
+  "scripts": {
+    "build-tokens": "paragon build-tokens --source ./paragon/tokens/ --build-dir ./paragon/build -t dark -t light",
```

Comment:
If we're assuming brand packages will be defining their tokens under `./paragon/tokens`, should we have that directory already created when this repo is used as a template for a custom brand package? For example, while this package wouldn't define any token overrides itself, we could have an empty `tokens` directory with a `.gitkeep` so it stays in the git history.